OpenEdX/Managers/DeepLinkManager/Link/PushLink.swift

```diff
@@ -21,10 +22,10 @@ public class PushLink: DeepLink {
     let body: String?
     
     override init(dictionary: [AnyHashable: Any]) {
-        let aps = dictionary[DataKeys.aps] as? [String: Any]
-        let alert = aps?[DataKeys.alert] as? [String: Any]
-        title = alert?[DataKeys.title] as? String
-        body = alert?[DataKeys.body] as? String
+        let aps = dictionary[DataKeys.aps.rawValue] as? [String: Any]
```

Comment:
Because of Anyhashable `RawStringExtractable` is not useable so please remove it from `DataKeys`.